webrtc.html

```diff
@@ -3782,8 +3805,8 @@
                 then perform the following checks:</p>
                 <ul>
                   <li>
-                    <p>If <var>t</var>'s <a data-for=
-                    "RTCRtpTransceiver">direction</a> is "sendrecv" or
+                    <p>If <var>t</var>'s [[<a href=
+                    "#direction-slot">Direction</a>]] slot is "sendrecv" or
```

Comment:
Created issue #1370 to track the question about code-tags or strings for enum values.